src/test/run-pass/union/union-align.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(attr_literals)]
+#![feature(repr_align)]
+#![feature(untagged_unions)]
```

Comment:
I looked more into why this is the case - it's needed because of the struct inside the union in one of the tests. The struct is not Copy by default and unions require their members to be copy by default. Non-copy types in untagged unions aren't stabilised yet. If I made the struct derive from Copy then I could probably remove the feature, I think I'll leave it for now though.